### DISCUSSION TITLE
Help needed with new features ZTS-EUB

### DISCUSSION THREAD

**Discussion by ciB89** [Koenkk/zigbee2mqtt#9695] [Devices]
I recently bought some Moes ZTS-EUB on aliexpress (aliexpress.com/item/1005002868874787.html). In z2m they are currently identified as ZTS-EU_1gang (_TZE200_amp6tsvy) - which seems to be wrong because they look completely different and mine have an optional neutral wire.

This new version has new feautres, for example "Tuya multi-control association" which can link multiple switches together to work as one (only works when neutral is connected). 
You can also disable/change the status led (which my wifey would approve of in the bedroom).

The device also currently does not act as a router, but I would imagine (maybe I am wrong) that when the neutral is connected, it should/could act as a router?

Since I am very inexperienced with zigbee and especially tuya devices, I would need some help in order to make these features for the device work. I have currently no tuya hub here, but it should arrive soon (which should help in the process as far as I understood the docs).

If there is anyone who would be willing to guide me through the process, I would highly appreciate it.

**Comment by Tonio16** [comment]
Hello
Did you achieved to have Tuya multi-control association with zigbee2mqtt?

Tx


Antoine

**Comment by SerialVelocity** [comment]
This is probably related to #13323
